landoapi/transplant_client.py

```diff
@@ -0,0 +1,90 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+import requests
+import requests_mock
+
+
+class TransplantClient:
```

Comment:
It's a to be default Transplant client. Currently mocked as we didn't wanted to make tha request yet.